lib/src/main/java/uk/gov/di/ipv/core/library/domain/PostCiMitigationRequest.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class PostCiMitigationRequest {
+    @SerializedName(value = "govuk_signin_journey_id")
+    private final String govukSigninJourneyId;
+
+    @SerializedName(value = "ip_address")
+    private final String ipAddress;
+
+    @SerializedName(value = "signed_jwt_list")
```

Comment:
We've named this `signed_jwts` in the CI storage request, but can easily update it if we want to

Reply:
👍  I'll update it here